app/src/androidTest/java/org/systers/mentorship/view/fragments/ChangePasswordFragmentTest.kt

```diff
@@ -0,0 +1,140 @@
+package org.systers.mentorship.view.fragments
```

Comment:
@justdvnsh this includes the commits for (a) scenario as well right? Remember to create separate branches for each issue you are working on. This helps in having separate commits for each issue.

Reply:
Hi . Yes, I made a separate branch for this issue, but that was what I was asking. My all new branches are having the commit from previous branches. I have made a PR for Logout Test as well, and it also contains these commits !!